### DISCUSSION TITLE
Issue Thread

### DISCUSSION THREAD

**Discussion by Hydraxous** [Hydraxous/UltraFunGuns#5] [Announcements]
Hello! Write about any issues you encounter here in the comments. It is helpful to include as much information as possible to help identify and fix the issue.
* Mod version
* Game version
* Operating system
* Use of any other mods
* What should have happened
* What happened instead
* Where did it happen
* What was happening in game when the issue was encountered
* Screenshots of the issue if applicable

**Comment by Primesoul1** [comment]
so i decided to install the 1.17 version of the mod and i have the game on steam the mods that im using are CGCustomwave and the EasyPZ mod but anyway what was supposed to happen was well be normal have textures and nothing like eggs floating or whatever but what happend was that the ball texture the flash texture for the focalyzer's the egg splatter are missing textures. please tell me how i can fix this.

**Reply by Hydraxous** [reply]
Hi there, sorry to hear things aren't working properly for you. I have a few follow up questions here.

1. What operating system are you using?
2. Do you have screenshots you can provide of the issue?
3. Have you already attempted a reinstall of the mod?

**Comment by Primesoul1** [comment]
1. Im using Windows 10
2. I don't have screenshots of the problem but i can take some screenshots.
3. Yes i have tried to reinstall the mod and i even tried to use diffrent
versions.

sob., 29 paź 2022, 00:52 użytkownik Hydraxous ***@***.***>
napisał:

**Comment by swirlyboii** [comment]
i cant open the inventory even if i press I or in the pause menu
im using windows 7
the mod version is 1.2.0
the game version is the latest version
i only have 6 mods
please help

**Reply by Hydraxous** [reply]
Version 1.2.0 is not compatible with violent night update or updates newer than violent night.
You can get the latest in https://GitHub.com/hydraxous/ultrafunguns/releases/latest

**Comment by Hydraxous** [comment]
Thread is closed in favor of the built-in issues feature.
Use that instead if you need to submit an issue.
